bin/install.sh

```diff
@@ -32,9 +32,16 @@ if [ -z "${JULIA_VERSION}" ]; then
     stop "JULIA_VERSION is not defined; don't know what to download"
 fi
 
-MAJOR="$(echo "${JULIA_VERSION}" | cut -d'.' -f 1)"
-MINOR="$(echo "${JULIA_VERSION}" | cut -d'.' -f 2)"
-PATCH="$(echo "${JULIA_VERSION}" | cut -d'.' -f 3)"
+if [[ $JULIA_VERSION == *.* ]]; then
+  MAJOR="$(echo "${JULIA_VERSION}" | cut -d'.' -f 1)"
```

Comment:
The rest of the repository uses 4 spaces for indentation, so it would be good to be consistent with that.

Reply:
Sorry I missed it.